compiler/dyno/include/chpl/parsing/FileContents.h

```diff
@@ -36,7 +40,7 @@ namespace parsing {
 class FileContents {
  private:
   std::string text_;
-  ErrorMessage error_;
+  const ParseError* error_;
```

Comment:
Could you add a TODO comment that it would be better to use the LLVM error handling strategy here?